src/scripts/generateApi.js

```diff
@@ -2,9 +2,10 @@ const fs = require("fs");
 const https = require("https");
 const lodash = require("lodash");
 
-const occUrl = process.env.OccUrl;
-const occToken = process.env.OccToken;
+const occUrl = "https://occ-dev-ocs-cm.develop.orckestra.cloud/api/openapi?removeDuplicatedBodyElements=true";
+const occToken = "mBTOMa98OOAtQ+dKqUJT50oGuqNoW87tbPPdLQf9KDkMxpx6ZCj7L99QlTRMbyCRNG2Vd6mVL7IO3VROWFxgnZonqtMKwM8alWrjvcnjRAg=";
```

Comment:
yeah......you just exposed one of our security keys to the world. You need to work with Martin to scrub (rewrite history) this git repo of this change

Reply:
for the future: you can put these values in a `.env` file